project.clj

```diff
@@ -1,11 +1,12 @@
-(defproject airtable-clj "0.1.0-SNAPSHOT"
```

Comment:
This file doesn't seem like it should've been changed as part of this pull request. Could you revert?

Reply:
This comment still applies.